roles/openshift_builddefaults/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Set builddefaults
+  openshift_facts:
+    role: builddefaults
+    openshift_env: "{{ hostvars[inventory_hostname]
+                       | oo_merge_dicts(hostvars)
+                       | oo_openshift_env }}"
+    openshift_env_structures:
+    - 'openshift.builddefaults.env.*'
```

Comment:
Indeed. Copy pasta.

On Friday, April 22, 2016, Jason DeTiberus <notifications@github.com> wrote:

> In roles/openshift_builddefaults/tasks/main.yml
> <https://github.com/openshift/openshift-ansible/pull/1385#discussion_r60781320>
> :
>
> > @@ -0,0 +1,17 @@
> > +---
> > +- name: Set builddefaults
> > +  openshift_facts:
> > +    role: builddefaults
> > +    openshift_env: "{{ hostvars[inventory_hostname]
> > +                       | oo_merge_dicts(hostvars)
> > +                       | oo_openshift_env }}"
> > +    openshift_env_structures:
> > +    - 'openshift.builddefaults.env.*'
>
> Should this just be 'openshift.builddefaults.*' instead?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1385/files/d9cb88f6da6272028bc773df4276202cd1810306..666d3649800a4586efe75d030a554c739accd0f9#r60781320>
>